src/applications/appeals/testing/sc/pages/optionIndicator.js

```diff
@@ -33,12 +31,15 @@ export default {
   },
   schema: {
     type: 'object',
+    required: [],
     properties: {
       'view:vhaContent': {
         type: 'object',
         properties: {},
       },
-      optionIndicator: radioSchema(Object.keys(optionIndicatorChoices)),
```

Comment:
This usage would be hard for consumers to pick up intuitively. It would be great if we could keep consistency with always using radioSchema. 
Is there some way we can make this work where the user can still use radioSchema?

Reply:
Hmm, the only thing I can think of would be passing in a boolean value to indicate that the radio group is required or not.

Reply:
For example:

```js
optionIndicator: radioSchema(Object.keys(optionIndicatorChoices), { required: false }),
```

Reply:
Added and used `radioOptionalUI` and `radioOptionalSchema` per your suggestions.